assets/scripts/aircraft.js

```diff
@@ -2151,6 +2156,11 @@ var Aircraft=Fiber.extend(function() {
 
       return ["ok", "proceed direct " + fixname];
     },
+    runDelete: function(data) {
+      for(var i=0;i<prop.aircraft.list.length;i++)
+ +      if(prop.aircraft.list[i].getCallsign() == this.getCallsign()) prop.aircraft.list[i].to_remove = true;
```

Comment:
@JHG0, these  2 lines should be simplified to `this.to_remove = true`, the command dispatch logic has already verified no other aircraft match this callsign.  After that I am ready to merge.